core/cap-0023.md

```diff
@@ -69,9 +69,7 @@ enum ClaimPredicateType
     CLAIM_PREDICATE_AND = 1,
     CLAIM_PREDICATE_OR = 2,
     CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME = 3,
-    CLAIM_PREDICATE_AFTER_ABSOLUTE_TIME = 4,
-    CLAIM_PREDICATE_BEFORE_RELATIVE_TIME = 5,
-    CLAIM_PREDICATE_AFTER_RELATIVE_TIME = 6
+    CLAIM_PREDICATE_BEFORE_RELATIVE_TIME = 4
```

Comment:
missing `CLAIM_PREDICATE_NOT`

Reply:
in fact enum is not what you put here, it's:
```
    CLAIM_PREDICATE_UNCONDITIONAL = 0,
    CLAIM_PREDICATE_AND = 1,
    CLAIM_PREDICATE_OR = 2,
    CLAIM_PREDICATE_NOT = 3,
    CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME = 4,
    CLAIM_PREDICATE_BEFORE_RELATIVE_TIME = 5
```

Reply:
Fixed